src/xprs_callbacks.jl

```diff
@@ -4,6 +4,9 @@ mutable struct CallbackData
     data::Any # data for user
     model::XpressProblem # local model # ptr_model::Ptr{Nothing}
 end
+Base.cconvert(::Type{Ptr{Cvoid}}, x::CallbackData) = x
+Base.unsafe_convert(::Type{Ptr{Cvoid}}, x::CallbackData) = x.ptr::Ptr{Cvoid}
```

Comment:
I don't think this is right. `x.ptr` doesn't exist.

Reply:
No it isn't. It should be
```julia
Base.unsafe_convert(::Type{Ptr{Cvoid}}, x::CallbackData) = x.model.ptr::Ptr{Cvoid}
```
But I think these lines aren't used in the code, so we could just remove them.

Reply:
Yeah just remove them